src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp

```diff
@@ -315,4 +315,44 @@ WellControls const & WellSolverBase::getWellControls( WellElementSubRegion const
   return this->getGroup< WellControls >( subRegion.getWellControlsName());
 }
 
+real64 WellSolverBase::setNextDt( real64 const & time, const real64 & lastDt, geos::DomainPartition & domain )
```

Comment:
I think we should include an option to disable this. I might very finely discretise my rate function without wanting geos to take such small time steps.

Reply:
ok sounds good

Reply:
Added `timeStepFromTables` flag to enable new behavior, disabled by default